devenv/scripts/deploy_private_e2e_tester.sh

```diff
@@ -0,0 +1,41 @@
+CLUSTER_RESOURCE_GROUP=$(cat state/cluster-info.json | jq '.ClusterResourceGroup' | tr -d '"')
+CLUSTER_NAME=$(cat state/cluster-info.json | jq '.ClusterName' | tr -d '"')
+
+export IMAGE=$(cat state/e2e-image-tag.txt)
+envsubst < e2e-tester.yaml > state/e2e-tester-formatted.yaml
+
+echo "deleting any existing e2e job..."
+az aks command invoke --resource-group $CLUSTER_RESOURCE_GROUP --name $CLUSTER_NAME --command "kubectl delete jobs app-routing-operator-e2e -n kube-system"
+
+set -e
+az aks command invoke --resource-group $CLUSTER_RESOURCE_GROUP --name $CLUSTER_NAME --command "kubectl apply -f e2e-tester-formatted.yaml" --file state/e2e-tester-formatted.yaml
+
+RESULT="null"
+while [ "$RESULT" = "null" ]
+do
+  STATUS=$(az aks command invoke --resource-group $CLUSTER_RESOURCE_GROUP --name $CLUSTER_NAME --command 'kubectl get pods --selector=app=app-routing-operator-e2e --output="jsonpath={.items[0].status.containerStatuses[?(@.name==\"tester\")].state}" -n kube-system' -o json | jq ".logs" | tr -d '\\')
+  STATUSNOQUOTES=${STATUS#"\""}
```

Comment:
commenting the prefix/suffix removal would make this clearer for anyone who isn't as famiar with bash